tally_ho/apps/tally/templates/audit/dashboard.html

```diff
@@ -6,11 +6,13 @@
 
 <h1>{% trans 'Audit List' %}</h1>
 
+{% if forms %}
```

Comment:
I think the button should display regardless, it can be disabled with a tooltip on why if there are no forms, but it's not good UX for buttons to disappear